src/order/copy.lean

```diff
@@ -17,17 +17,26 @@ universe u
 
 variables {α : Type u}
 
-/-- A function to create a provable equal copy of a bounded lattice
+/-- A function to create a provable equal copy of a bounded order
 with possibly different definitional equalities. -/
-def bounded_lattice.copy (c : bounded_lattice α)
-  (le : α → α → Prop) (eq_le : le = @bounded_lattice.le α c)
-  (top : α) (eq_top : top = @bounded_lattice.top α c)
-  (bot : α) (eq_bot : bot = @bounded_lattice.bot α c)
-  (sup : α → α → α) (eq_sup : sup = @bounded_lattice.sup α c)
-  (inf : α → α → α) (eq_inf : inf = @bounded_lattice.inf α c) :
-  bounded_lattice α :=
+def bounded_order.copy [has_le α] (c : bounded_order α)
```

Comment:
This is weaker than the original, it doesn't let you copy a bounded order to a different has_le instance.

Ideally this would take a second `has_le` argument and an `le_eq` proof